packages/components/src/slot-fill/utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import {
+	isArray,
+	isNumber,
+} from 'lodash';
+
+export const isEmptyElement = ( element ) => {
```

Comment:
We use it only here, if there is going to be another use case, we could think to move it `@wordpress/element`.